Makefile

```diff
@@ -14,7 +14,7 @@ AUDIT_CONNECTION ?= "audit"
 AUDIT_CHANNEL ?= "audit"
 LOG_LEVEL ?= "INFO"
 
-VERSION := v3.16.0-beta.0
+VERSION := v3.17.0-beta.0
```

Comment:
why 17?

Reply:
good catch! missed this because of local testing. Ty!